webapp/src/test/java/com/box/l10n/mojito/service/thirdparty/ThirdPartyServiceTest.java

```diff
@@ -130,6 +130,16 @@ void initThirdPartyTMSMock() {
         .createImageToTextUnitMappings(any(), any());
   }
 
+  @Test
+  public void testRemoveImage() {
+    String projectId = "projectId";
+    String contextId = "contextId";
+
+    thirdPartyTMSMock.removeImage(projectId, contextId);
```

Comment:
what was the intent here? Unless I'm missing a detail, this is not testing anything.

Reply:
line 140 is testing if the call tp the underlying `thirdPartyTMSMock.removeImage()` was correctly made.